doc/gen_imgs.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+{
+  ver=2.x
+
+  ${JULIA-julia} gen_imgs.jl $ver
+
+  for f in imgs/$ver/*.txt; do
+    html=${f%.txt}.html
+    cat $f | ${ANSI2HTML-ansi2html} --input-encoding=utf-8 --output-encoding=utf-8 >$html
+    ${WKHTMLTOIMAGE-wkhtmltoimage} --quiet --crop-w 800 --quality 85 $html ${html%.html}.png
```

Comment:
Good to know this 👍

Reply:
This is hacky, but was the only sequence that allowed me to generate correct pngs.

`ansi2html` does a good job, but then the issue is to include `html` in a `markdown`, and I haven't found a way to do that, so I've defaulted to `png` conversion.